config/Config.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+common_caps = {
+    "browserstack.user" : "BROWSERSTACK_USERNAME",
+    "browserstack.key" : "BROWSERSTACK_ACCESS_KEY",
+    "build" : "browserstack-build-1",
+    "browserstack.debug" : "true"
+}
+
+envs = [{
+    "os" : "Windows",
+    "os_version" : "10",
+    "browser" : "Chrome",
+    "browser_version" : "latest"
+},
+{
+    "os" : "OS X",
+    "os_version" : "Big Sur",
+    "browser" : "Safari",
+    "browser_version" : "latest"
+},
+{
+    "device" : "Samsung Galaxy S22",
+    "os_version" : "12",
+    "real_mobile" : "true"
```

Comment:
```suggestion
    "os_version" : "12"
```

Reply:
real_mobile is true (also false is not honoured) by default so we can remove this